database/batch.go

```diff
@@ -14,39 +14,31 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-package ethdb
+package database
 
-// Code using batches should try to add this much data to the batch.
-// The value was determined empirically.
+// IdealBatchSize defines the size of the data batches should ideally add in one
+// write.
```

Comment:
Out of curiousity: was this number determined to be system-agnostic? It might be worth adding that in the comment to help provide clarity.